concourse/pipelines/templates/jobs/test-backward-compatibility-tpl.yml

```
@@ -0,0 +1,55 @@
+{# Template for a PXF backwards compatibility job definition.
+  #
+  # Expects a dictionary with the name 'x' to be set as defined in macros.j2 file and these additional fields:
+  #
+  # x.num_versions    - the number of GPDB versions to test against
+  #}
+{% if x.build_platform == x.test_platform %}
+
+{% set name = 'test-pxf-gp' ~ x.gp_ver ~ '-backward-compatibility-' ~ x.test_platform %}
```

even though the `{%` cannot be indent, but we can indent inside the `{%`, e.g. 
```suggestion
  {% set name = 'test-pxf-gp' ~ x.gp_ver ~ '-backward-compatibility-' ~ x.test_platform %}
```

---

as per discussion, jinja spacing changes will be pushed off to a different chore/PR